src/vivarium/framework/lookup/manager.py

```diff
@@ -204,7 +207,7 @@ def validate_build_table_parameters(
     ):
         raise ValueError("Must supply some data")
 
-    acceptable_types = (Numeric, datetime, timedelta, list, tuple, pd.DataFrame)
+    acceptable_types = (Numeric, datetime, timedelta, list, tuple, pd.DataFrame, dict)
```

Comment:
Data will have been converted from a dict to a Dataframe in line 89, so we don't want to add it here.